robottelo/ui/contentviews.py

```diff
@@ -233,14 +234,14 @@ def add_remove_repos(self, cv_name, repo_names, add_repo=True):
             self.find_element(tab_locators["contentviews.tab_content"]).click()
             self.find_element(locators["contentviews.content_repo"]).click()
             self.wait_for_ajax()
-            if add_repo:
-                self.find_element(tab_locators
-                                  ["contentviews.tab_repo_add"]).click()
-            else:
-                self.find_element(tab_locators
-                                  ["contentviews.tab_repo_remove"]).click()
             strategy, value = locators["contentviews.select_repo"]
             for repo_name in repo_names:
+                if add_repo:
+                    self.find_element(tab_locators
+                                      ["contentviews.tab_repo_add"]).click()
+                else:
+                    self.find_element(tab_locators
+                                      ["contentviews.tab_repo_remove"]).click()
```

Comment:
This is the existing code just moved this code below. 
So, I just copy pasted the existing code.